build.gradle

```diff
@@ -20,7 +20,7 @@ project.ext {
     slf4jVersion = '1.7.25'
     mockitoVersion = '1.9.5'
     assertjVersion = '3.8.0'
-    zkToolsVersion = '0.6.0'
+    zkToolsVersion = '0.7.0'
```

Comment:
ZooKeeperClientImpl() is updated in 0.7.0:
https://github.com/wepay/zktools/commit/62cd5ccabdceaa5abebb1c57e8c65e2073d2fc1f#comments